content/en/docs/js/_index.md

```diff
@@ -4,6 +4,9 @@ weight: 1
 ---
 
 This page contains an introduction to OpenTelemetry in JavaScript. This guide will walk you through installation and instrumentation and show you how to export data.
+=======
+This page contains an introduction to OpenTelemetry in JavaScript. This guide will walk you through installation and instrumentation and show you how to export data.
+>>>>>>> b26cf3c21f64a32d97310d097965e54147f24cb6
```

Comment:
Bad merge? This should be cleaned up

Reply:
Ugh... yes, let me see if I can fix that or if I just do a fresh start.

Reply:
Remove that one, thanks